README.md

```diff
@@ -55,25 +65,38 @@ DynatraceMetricExporter exporter =
 After acquiring a `DynatraceMetricExporter` object, it has to be registered with the OpenTelemetry SDK using a `MetricReader`:
 
 ```java
+import java.time.Duration;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.metrics.export.PeriodicMetricReader;
+
 // Create the MeterProvider and register it globally. 
 // The MeterProvider is configured with the PeriodicMetricReader
 // which takes our exporter and the export interval.
 SdkMeterProvider meterProvider =
     SdkMeterProvider.builder()
         .registerMetricReader(
-            PeriodicMetricReader.builder(exporter).setInterval(Duration.ofSeconds(60)).build())
+            PeriodicMetricReader.builder(exporter).setInterval(Duration.ofSeconds(1)).build())
```

Comment:
I think we usually recommend 10-60 secs. If one copy-pastes it as-is (and misses the new comment below), then 10 secs might be a good compromise between seeing results quickly and not exporting more often than reasonable.

Reply:
I guess I can move to 10. But waiting ~15 seconds to try it out is not so great I think. And if they copy paste 10 seconds into production, it's also bad I think 😓. I could maybe add the comment directly here to make it at least more apparent

Reply:
I kept 1 for interval and added the comment also here.